build/pythontool.py

```diff
@@ -1,10 +1,15 @@
 from waflib import Options
-import os, re
+import os, sys, re
 
 def options(opt):
     opt.load('python')
-    opt.add_option('--disable-python', action='store_false', dest='python',
-                   help='Disable python', default=True)
+    if sys.platform == 'sunos5':
+        opt.add_option('--enable-python', action='store_true',
+                dest='python', help='Disable python', default=False)
```

Comment:
Done.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Thursday, May 05, 2016 8:38 AM
To: mdaus/coda-oss <coda-oss@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [mdaus/coda-oss] Fix Solaris build by disabling Python and replacing deprecated os.pop… (#116)


In build/pythontool.py<http://cp.mcafee.com/d/FZsSd39J5xNdxZ4QsEzztPrb38UsqenTPrUVBxB4QsCzASztWXPxEVKqenzqabaarVEVsovdydorcDIel_ok1iG6JQVsSGxHtendSJX91cs_R-pjd79EV73DKLsKCOevVNxMsYPvBHEShhlKUDOEuvkzaT0QSyrjdTV5MQsL8EIFTudTdAVPmEBC4pj9JAenNl3mWpqG6BWQFyPpV-8EK0oxaBCWkBm0H5Nd5WUVVcSgGT2TQ02YMYMyOnpohQK8RxIOuMVnZxgVl3mWsK2G4hfycFASO7bCTS4TzobZ8Qg6BJl3mWpqG6BWQEqnjh0bO3Ph0clfScG6y0wT8Z0Qg42NEwgBji1YE4jh05yQOto6y3o86y2fH7Xqk29EwgZqfSSvE6y2JemGIqUzk0qFzh0qa0chhsd59J6ZPtHND>:

>

>  def options(opt):

>      opt.load('python')

> -    opt.add_option('--disable-python', action='store_false', dest='python',

> -                   help='Disable python', default=True)

> +    if sys.platform == 'sunos5':

> +        opt.add_option('--enable-python', action='store_true',

> +                dest='python', help='Disable python', default=False)

help should be "Enable python"

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/1jWVIg40Up43qb3yr3W9EVh76XCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKrJrSi2oV_HYOCqejhOe7ftuVtdAs_Pz3wVVC_bnhIyyHtNfBgY-F6lK1FJcSCrLObxEVuhhpjKYrKr9PCJhbc8OCjr8sLyG6JQORkdbRFj5CPPYhhsNSh_QMse_XKmd7ababdL76zB7mmnjhOMPtWP391BBRNPrent0ObyqbRNPOpIxlK5LE05VxVx5AKOMzFshH3pAZxOLX2xOG6JQVs5k8yv4pj9JAendLI9L6MnWhEwdbqG6JQORkdbRFgQKCy0nA7Cy0oGvIpkd411KhW1Ew85zh0xaCA3Vg8Cy0b5FAWMd46Mgd44vmfSQE4jh0xWQvJI_gd45qsJloRN6E0Rj6y0Qk0oyyUqajqdXC-We8>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.